nucliadb_texts2/tests/test_search.rs

```diff
@@ -97,13 +98,14 @@ fn test_prefilter_not_search() {
         paragraph_labels: HashSet::with_capacity(0),
     };
     let query = "{ \"not\": { \"literal\": \"/l/mylabel\" } }";
-    let expression = nucliadb_core::query_language::translate(query, &context).unwrap();
+    let expression = nucliadb_core::query_language::translate(Some(query), None, &context).unwrap();
```

Comment:
Could be nice to add some test in the Rust side too, instead of setting the keyword filter always to `None`